.eslintrc.js

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** @type { import('@typescript-eslint/utils').TSESLint.Linter.Config } */
+/** @type { import('@typescript-eslint/utils/dist/index').TSESLint.Linter.Config } */
```

Comment:
Not really sure why this happened. The exports of `@typescript-eslint/utils` looks alright, and points to `dist/index` but for some reason we still can't import it directly, need to use a sub directory import... Not really worth digging into why exactly 🤷